src/scripts/main.coffee

```diff
@@ -1063,6 +1072,9 @@ class Formbuilder
         APPROVER_ID: 'options.approver_id'
 
     change:
+      INLINE_IMAGES_ENABLED: ->
+        @reset()
+        @resetInlineImages();
```

Comment:
Also harmless I think but you shouldn't need the `;`, will be compiled into JS with one, just for conformity.